bolt-contracts/config/holesky/operator.json

```diff
@@ -1,5 +1,6 @@
 {
-    "rpc": "localhost:50051",
-    "salt": "0x000000000000000abc0000000000000000000000000000000000000000000000",
-    "expiry": null
-}
\ No newline at end of file
+  "rpc": "<hostname>:<port>",
+  "salt": "0x0000000000000000000_salt_value_0000000000000000000000000000000000",
+  "expiry": "0x00000000000000000_expiry_value_0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Same here. Expiry is the hex-encoding of a UNIX timestamp in seconds. Can convert it to hex with `printf "%x\n" $TIMESTAMP_DECIMALS`